text/0000-kpack-donation-to-cnb.md

```diff
@@ -0,0 +1,374 @@
+
+#### RFC process
+
+Once the migration is completed, [kpack](https://github.com/pivotal/kpack/) will follow the [RFC process](https://github.com/buildpacks/rfcs) stablished in CNB project for any new RFC created in the project.
+
+##### Existing RFC
+
+- **Open**: Currently there are less that 10 [open RFCs](https://github.com/pivotal/kpack/pulls?q=is%3Apr+label%3ARFC+is%3Aopen) (some of them opened 2 years ago) in [kpack](https://github.com/pivotal/kpack/) repository.
+  - The proposal is to suggest the [kpack](https://github.com/pivotal/kpack/) maintainers to:
+    - Triage those RFCs an update their status before the donation.
+    - Co-ordinate the announcement of the donation to the RFCs authors and explain them the strategy after the migration (next section)
+    - After the donation, any open RFCs in [kpack](https://github.com/pivotal/kpack/) repository should be closed
+    - The RFC author should create a new RFC in the CNB RFC [repository](https://github.com/buildpacks/rfcs) and follow the CNB [RFC process](https://github.com/buildpacks/rfcs)
+
+- **Closed**: For historical purpose, we will keep those RFC in the repository.
+
+##### Template
+
+The following table shows the mapping between the sections of the current kpack [RFC template](https://github.com/pivotal/kpack/blob/main/rfcs/0000-template.md) vs the CNB [RFC template](https://github.com/buildpacks/rfcs/blob/main/0000-template.md)
+
+|   kpack template |   CNB template   |
+| ----------------------- | ------------------------------ |
+| Problem           | Motivation       |
+| Outcome           | Motivation       |
+| Actions to take   | What it is / How it works       |
+| Complexity           |    -   |
+| - | Migration |
+| Prior Art           | Prior Art       |
+| Alternatives          | Alternatives     |
+| Risks           | Drawbacks      |
+| - | Unresolved Questions |
+| - | Spec. Changes  |
+| - | History|
+
+As we can see the CNB [RFC template](https://github.com/buildpacks/rfcs/blob/main/0000-template.md) consider most of the sections required by the kpack [RFC](https://github.com/pivotal/kpack/blob/main/rfcs/0000-template.md) process, the only missing one is:
+  - **complexity**: What thoughts do you have on the complexity of this?
+
+The proposal is to incorporate this section into the [CNB template](https://github.com/buildpacks/rfcs/blob/main/0000-template.md).
+
+#### Slack channel
+
+The proposal is to move the [kpack](https://github.com/pivotal/kpack/) slack instance from the [Kubernetes slack instance](https://kubernetes.slack.com/channels/kpack) to the [CNCF slack instance](https://slack.cncf.io/).
```

Comment:
Many users of kpack are within kubernetes and may find it easier to communicate within the kubernetes slack. Is there any reason to not just have both channels?

Reply:
The only reason I can see its just something related to the maintenance of the account in Kubernetes slack, Platform maintainers should be able to have admin rights on that channel, right? 

@samj1912 will be ok if we propose to just create a new channel on CNCF slack?

Reply:
It should be okay to have channels in both places. I have seen other projects (like Kyverno) do that to ensure coverage. Just would be nice to decide on one canonical/preferred channel for slack communications.

Reply:
Well, that's ok, I think the preferred channel must be the new one created (under CNCF slack instance) as that's how we [tell](https://buildpacks.io/community/) everybody to find us in the website